packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -323,7 +323,7 @@ abstract class BindingBase {
           await setter(parameters['enabled'] == 'true');
           _postExtensionStateChangedEvent(name, await getter() ? 'true' : 'false');
         }
-        return <String, dynamic>{ 'enabled': await getter() ? 'true' : 'false' };
+        return <String, dynamic>{'enabled': await getter() ? 'true' : 'false'};
```

Comment:
seems like we're not even consistent in our style guide about this 

between https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements and the header right below it

Reply:
I'm not sure to understand what you mean.